### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by nerd190** [jakobrs/nix-termux#3]
Hello again!
I just reset my phone and I am now unable to install, I get this error:
```bash
/proc/self/fd/63: sorry, there is no binary distribution of Nix for your platform.
```
I followed the ReadMe like usual so something must've changed?
I read through everything and in setup you mention exporting boehtm to 1, is it this value has changed now in nix? or perhaps the nix installation script itself has changed?
Thank you once again for this, I really love this lightweight, language-agnostic distro!
EDIT: checked their repo and the install script was updated a few times since your last update 👍

**Comment by jakobrs** [comment]
That's interesting. What does `uname -s` and `uname -m` say?

**Comment by nerd190** [comment]
```bash
~ $ uname -s
Linux
~ $ uname -m
armv8l
```
Device: OnePlus 5T
OS: Lineage OS
Version: Android 10

Could it be because my device reports `armv8l` rather than the more common `aarch64`?

**Comment by jakobrs** [comment]
Yes, it's definitely because `uname -m` returns `armv8l` instead of `aarch64`.

I believe `armv8l` means it's running in 32-bit mode. Perhaps you've installed a 32-bit version of Termux by mistake? Or, depending on how you've reset your phone, you might have installed a 32-bit version of your phone's OS by mistake.

**Comment by nerd190** [comment]
Sorry for not closing this sooner!
Turns out it was a combination of:
- **Big** changes to Magisk...
- Custom kernel not being updated to reflect Magisk's changes.

Had to reset my phone back to stock and start fresh
(luckily, I found my original backup of completely stock Android!)

Apologies for opening this issue, I assumed as there were changes nix end and because this is the first time (since rooting my original Galaxy S1 ten years ago!) I have experienced this problem.

Side note: This is all due to Android turning more and more into an iPhone every year, so I've decided to stay here (on Oreo (8.1.0)) and not update to 9/10/11 where everything is restricted. Bliss!

Again, thank you so much, being able to carry Nix in my pocket is **really** useful, especially on my tablet.
